src/routes/reference/components/show.mdx

```diff
@@ -47,4 +47,4 @@ If the `keyed` property is not used, the argument of a child function will be an
 | :--------- | :-------------------------------- | :-------------------------------------------- |
 | `when`     | `T \| undefined \| null \| false` | The value to test for truthiness              |
 | `keyed`    | `boolean`                         | Whether to key the block to the value of when |
-| `fallback` | `JSX.Element`                     | The fallback to render when when is falsy     |
+| `fallback` | `JSX.Element`                     | The fallback to render when the `when` is falsy     |
```

Comment:
```suggestion
| `fallback` | `JSX.Element`                     | The fallback to render when `when` is falsy     |
```

This should either be `` `when` `` (no `the`) or `` the `when` condition `` (or `` the `when` value ``).